pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "jwave"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
This can be kept at `0.1.2`, the github action takes care automatically to upgrade the version at each release.